src/lib.rs

```diff
@@ -1,9 +1,85 @@
+#![allow(rustdoc::private_intra_doc_links)]
+/*!
+
+Easy to use profiler scopes for [wgpu](https://github.com/gfx-rs/wgpu) using timer queries.
+
+wgpu_profile manages all the necessary [wgpu::QuerySet] and [wgpu::Buffer] behind the scenes
```

Comment:
This is a matter of your preference, but a common style I see is wrapping types and crate names in backticks (even if they are also links).

e.g. 
```
`wgpu_profiler`
[`wgpu::QuerySet`]
[`wgpu::Buffer`]
```